fix(backups): handles task end in CR without health check (#6866)
vatesfr/xen-orchestra: @xen-orchestra/backups/docs/VM backups/README.md

    - `prepare({ isFull })`
    - `transfer({ timestamp, deltaExport, sizeContainers })`
    - `cleanup()`

>     - `healthCheck()` // may not be executed if no health check sr or tag doesn't match

Reviewer: ```suggestion
    - `healthCheck()` // is not executed if no health check sr or tag doesn't match
```
